ArrayIndexOutOfBoundException when running the analyser with AEM Release -2021.8.5705.20210810T082946Z-210700-a

The below exception occurs when analysing an AEM project with AEM Release 2021.8.5705.20210810T082946Z-210700

Caused by: java.lang.ArrayIndexOutOfBoundsException: arraycopy: source index -1 out of bounds for char[65536]
    at java.lang.System.arraycopy (Native Method)
    at org.apache.johnzon.core.JsonStreamParserImpl.doAutoAdjust (JsonStreamParserImpl.java:197)
    at org.apache.johnzon.core.JsonStreamParserImpl.appendToCopyBuffer (JsonStreamParserImpl.java:164)
    at org.apache.johnzon.core.JsonStreamParserImpl.readString (JsonStreamParserImpl.java:623)
    at org.apache.johnzon.core.JsonStreamParserImpl.handleQuote (JsonStreamParserImpl.java:694)
    at org.apache.johnzon.core.JsonStreamParserImpl.internalNext (JsonStreamParserImpl.java:452)
    at org.apache.johnzon.core.JsonStreamParserImpl.internalNext (JsonStreamParserImpl.java:412)
    at org.apache.johnzon.core.JohnzonJsonParserImpl.next (JohnzonJsonParserImpl.java:50)
    at org.apache.johnzon.core.JsonReaderImpl.parseObject (JsonReaderImpl.java:170)
    at org.apache.johnzon.core.JsonReaderImpl.parseObject (JsonReaderImpl.java:182)
    at org.apache.johnzon.core.JsonReaderImpl.parseObject (JsonReaderImpl.java:182)
    at org.apache.johnzon.core.JsonReaderImpl.parseObject (JsonReaderImpl.java:182)
    at org.apache.johnzon.core.JsonReaderImpl.readValue (JsonReaderImpl.java:89)
    at org.apache.johnzon.core.JsonReaderImpl.read (JsonReaderImpl.java:67)
    at org.apache.johnzon.core.JsonReaderImpl.readObject (JsonReaderImpl.java:141)
    at org.apache.sling.feature.io.json.FeatureJSONReader.readFeature (FeatureJSONReader.java:676)
    at org.apache.sling.feature.io.json.FeatureJSONReader.read (FeatureJSONReader.java:77)

--- a/aemanalyser-maven-plugin/pom.xml
+++ b/aemanalyser-maven-plugin/pom.xml
@@ -124,7 +124,7 @@ governing permissions and limitations under the License.
     <dependency>
       <groupId>org.apache.johnzon</groupId>
       <artifactId>johnzon-core</artifactId>
-      <version>1.2.10</version>
+      <version>1.2.14</version>
     </dependency>
     <dependency>
       <groupId>org.apache.felix</groupId>

